src/components/molecules/widgets/CountInjuredByYearBarWidget.tsx

```diff
@@ -6,11 +6,15 @@ import { MultiBarChart } from '../GenericBarChart';
 
 interface IProps {
   data: IWidgetMultiBarData;
+  barOptions: any;
 }
 
-const CountInjuredByYearBarWidget: FC<IProps> = ({ data }) => {
+const CountInjuredByYearBarWidget: FC<IProps> = ({ data, barOptions }) => {
   const { items, text } = data;
-  const multiBarSeries = convertToBarSeries(items, text.labels_map);
-  return <MultiBarChart isStacked={true} isPercentage={false} data={multiBarSeries} textLabel={text.title} />;
+  const excludeList = barOptions ? Object.values(barOptions).map((include: any, index: number) => {
+    if (include) {return null} else {return items[0].series[index].label_key}}) : []
```

Comment:
if it has the same logic like in `CountInjuredByYearBarWidget.tsx` please extract this to as a helper function in the proper file

Reply:
What is the proper file for this?

Reply:
you can use `barChart.utils.ts`